usage.txt

```diff
@@ -26,6 +26,9 @@ out                the dir to put the app into at the end. defaults to current w
 overwrite          if output directory for a platform already exists, replaces it rather than skipping it
 prune              runs `npm prune --production` on the app
 sign               should contain the identity to be used when running `codesign` (only for building for the darwin platform, on OS X)
+appCategoryType    set application category type.
```

Comment:
This is the only command line parameter that's camel cased. Is it possible for it to be hyphenated instead?